modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -240,10 +236,22 @@ public void processOpts() {
         supportingFiles.add(new SupportingFile("__init__package.mustache", packagePath(), "__init__.py"));
         supportingFiles.add(new SupportingFile("__init__model.mustache", packagePath() + File.separatorChar + modelPackage, "__init__.py"));
         supportingFiles.add(new SupportingFile("__init__api.mustache", packagePath() + File.separatorChar + apiPackage, "__init__.py"));
+
+        // If the package name consists of dots(openapi.client), then we need to create the directory structure like openapi/client with __init__ files.
+        String[] packageNameSplits = packageName.split("\\.");
+        String currentPackagePath = "";
+        for (int i = 0; i < packageNameSplits.length-1; i++) {
+            if (i > 0) {
+                currentPackagePath = currentPackagePath + File.separatorChar;
+            }
+            currentPackagePath = currentPackagePath + packageNameSplits[i];
+            supportingFiles.add(new SupportingFile("__init__.mustache", currentPackagePath, "__init__.py"));
+        }
+
         supportingFiles.add(new SupportingFile("exceptions.mustache", packagePath(), "exceptions.py"));
 
         if (Boolean.FALSE.equals(excludeTests)) {
-            supportingFiles.add(new SupportingFile("__init__test.mustache", testFolder, "__init__.py"));
+            supportingFiles.add(new SupportingFile("__init__.mustache", testFolder, "__init__.py"));
```

Comment:
@saigiridhar21 can you share more on changing __init_test.mustache to __init__.mustache?

Reply:
@wing328  In [line 248](https://github.com/OpenAPITools/openapi-generator/pull/2992/files#diff-519f453fe7e545fd0084af74b24bb90dR248), I need an empty `_init__.mustache`. Here, we are already having an empty `__init__test.mustache`. So, instead of having two empty `__init__` mustache files, I renamed `__init__test.mustache` to `__init__.mustache` and utilized it at both the places.

Reply:
Thanks for the explanation. Didn't realise we've 2 templates for empty `__init__`